scripts/push-vellum-workflows.ts

```diff
@@ -30,6 +31,75 @@ class VellumWorkflowPusher {
     this.isPreviewMode = isPreviewMode;
```

Comment:
Instead of accepting a preview mode param, infer whether this is a preview environment based on whether this is a pr branch or main